src/services/hooks/use-theme.tsx

```diff
@@ -0,0 +1,11 @@
+import { useLayoutEffect, useState } from "react";
+
+export const useTheme=()=>{
+  const[ theme, setTheme ] = useState(localStorage.getItem("select-theme") || "light");
+  useLayoutEffect(()=>{
+    document.documentElement.setAttribute("data-theme", theme);
```

Comment:
please, try to find another possibility to do that - without using `document` variable (it's badly to use it at the react)